cmd/agent/entrypoint.go

```diff
@@ -250,17 +262,21 @@ func (ep *Entrypoint) TriggerReload() bool {
 
 	cfg, err := ep.reloader()
 	if err != nil {
+		ep.configReloadSuccess.Set(0)
```

Comment:
Also in thinking about this I wonder if we need to change it to just `load`, or set values to good on startup. If I am alerting based on these they will not be valid til a reload happens.

Reply:
Shouldn't the default be success? This is because the "first" load of the config is a success and if its bad config, the startup itself will fail. I would default to success tbh.

Reply:
uhm, let's move this to config.go so that we keep track of loads and not just reloads